src/roswire/common/source.py

```diff
@@ -29,100 +32,70 @@ class SourceLanguage(enum.Enum):
     PYTHON = "python"
 
 
-@attr.s(frozen=True, auto_attribs=True, slots=True)
-class NodeSourceInfo:
-    node_name: str
+class ExecutableKind(enum.Enum):
+    NODE = "node"
+    LIBRARY = "library"
```

Comment:
A library isn't really a kind of executable, but both binaries and libraries are considered _build targets_.